ci-operator/step-registry/cucushift/agent/vsphere/disconnected/provision/cucushift-agent-vsphere-disconnected-provision-chain.yaml

```diff
@@ -22,7 +22,7 @@ chain:
       documentation: |-
         Register DNS for mirror registry.
     - name: BASTION_HOST_SUBNET
-      default: "qe-segment"
+      default: "devqe-segment-221"
```

Comment:
as you already change `BASTION_HOST_SUBNET` to segment on new vsphere, I guess that cluster profile also needs to be adjusted to `vsphere-dis` in relevant job and job config?

Reply:
Yes, I'll move all job with profile vsphere-discon to vsphere-dis

Reply:
okay, after that, I think we can also verify the workflow `cucushift-installer-rehearse-vsphere-upi-disconnected-workers-rhcos-rhel8` in this PR, but need to update `RHEL_IMAGE` to `qe-rhel-88-release` in that install chain.